examples/keras_io/nlp/text_classification_with_transformers.py

```diff
@@ -0,0 +1,115 @@
+"""
+Title: Text Classification with Transformers
+Author: [Yash Srivastava](https://twitter.com/Yaaaaaashhh)
+Date created: 2023/07/16
+Last modified: 2023/07/16
```

Comment:
If this is not written from scratch, I guess the original date needs to be kept. Check: https://github.com/keras-team/keras-core/pull/491#discussion_r1264493629

Reply:
Ok. Sure. I'll change this.